src/atomate2/forcefields/schemas.py

```diff
@@ -104,6 +127,21 @@ class ForceFieldTaskDocument(StructureMetadata):
         None, description="Directory where the force field calculations are performed."
     )
 
+    included_objects: Optional[list[ForcefieldObject]] = Field(
+        None, description="list of forcefield objects included with this task document"
+    )
+    forcefield_objects: Optional[dict[ForcefieldObject, Any]] = Field(
+        None, description="Forcefield objects associated with this task"
+    )
+
+    is_force_converged: Optional[bool] = Field(
+        None,
+        description=(
+            "Whether the calculation is converged with respect "
+            "to interatomic forces."
+        ),
+    )
+
     @classmethod
     def from_ase_compatible_result(
```

Comment:
Can you confirm that none of these changes impacts the compatibility between DFT and FF schemas in terms of the trajectory? E.g., hopefully we can use both DFT and FF trajectories interchangeably.

Reply:
Seconding, as a community member just adding a comment that any efforts to keep schemas stable and un-changing (backwards compatible) are hugely appreciated 🙏

Reply:
I'll go through and see if there are any incompatibilities. @QuantumChemist  seems to have found a missing `dir_name` which they patched in [#791](https://github.com/materialsproject/atomate2/pull/791), so there may be other missing attrs